.github/workflows/pove.yml

```
@@ -0,0 +1,92 @@
+        with: { version: nightly }
+      - name: Build & test
+        working-directory: ${{ needs.fail_fast.outputs.root }}
+        run: |
+          set -euo pipefail
+          forge fmt --check
+          forge build -vvv
+          forge test -vvv
+
+  deploy_amoy:
+    needs: build_test
+    if: ${{ github.event_name == 'workflow_dispatch' && secrets.PRIVATE_KEY != '' && secrets.RPC_AMOY != '' }}
```

actionlint (1.7.8)

context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)